podcasts/AppDelegate+UrlHandling.swift

```diff
@@ -45,7 +45,11 @@ extension AppDelegate {
                 alert.addAction(UIAlertAction(title: "Import", style: .default) { _ in
                     Task {
                         let fileWrapper = try FileWrapper(url: url)
-                        try PCBundleDoc.performImport(from: fileWrapper)
+                        do {
```

Comment:
Would it make sense to include `let fileWrapper = try FileWrapper(url: url)` in `do { ... }` to catch also any possible error?